### ISSUE TITLE
[Feature]: add new cluster input endpoint

### ISSUE THREAD

**Issue by kitsudaiki** [kitsudaiki/ainari#203]
### Description

At the moment there are 2 ways to insert data for training and requesting:

1. upload a data-set and use this as task on the cluster
2. use a websocket-connection to directly connect to the cluster

It would be nice, if it would be possible to send single train- and request-messages, similar to the websocket-connection, ober the REST-API. It would be much slower, but easier to use, which could be quite convenient in some cases.

### Acceptance Criteria

1. A new endpoint was added to train and test a cluster in direct-mode
2. New tests were added to the SDK-API-Tester

### Blocked by

- #418

**Comment by kitsudaiki** [comment]
already done in context of issue:  https://github.com/kitsudaiki/ainari/issues/529
